app/src/main/java/gateway/soap/request/MoveFileReq.java

```diff
@@ -0,0 +1,11 @@
+package gateway.soap.request;
+
+import java.util.UUID;
+
+public class MoveFileReq extends Operation
+{
+	public UUID nameUUID;
+	public UUID targetLocationUUID; // always a directory
+	// TODO: Add nullable flag
```

Comment:
Since `Java` apparently doesn't need to specify which field is `nullable` this comment can be removed